comp230_01_1607539.tex

```diff
@@ -0,0 +1,65 @@
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+\usepackage{graphicx}
+\usepackage{url}
+\usepackage[autostyle]{csquotes}
+
+% Please include a clear, concise, and descriptive title
+\title{What are the risks associated with nurturing a modding community, and how can game developers protect themselves?}
+
+% Please do not change the subtitle
+\subtitle{COMP230}
+
+% Please put your student number in the author field
+\author{Student number: 1607539}
+
+\begin{document}
+
+\maketitle
+
+Game developers can encourage modifications by releasing the game's source code to the public. An often repeated example is the case of \textit{Doom} (1993) in which the media files were separated from the main program and made accessible for users \cite{sotamaa2010game}. This is historically celebrated as a consequence of mutual respect \cite{hong2013game}. Modders use these toolsets to build mods of a number of different scales, from the massive ``total conversions'' accomplished by teams of modders who remake entire games with new artwork, storylines, and gameplay to smaller mods such as ``skin'' (aesthetic tweaks to specific graphic models) \cite{hong2014becoming}.  Generally going far beyond what is designed by the commercial developers \cite{postigo2007mods}.
+
+This paper will outline the risks of an active modding community, the benefits, and give examples of how developers have protected themselves.
+
+\subsection*{Modding culture}
+
+The motivation to produce a mod can vary from purely aesthetic to a more political one. Projects focusing on particular historical events (\textit{Operation Gulf War Crisis, Battle over Hokkaido, 1982: Flashpoint in the Falklands}) often have a national emphasis. Understandably these kinds of themes tend to open up political debates among mod makers \cite{sotamaa2010game}, which may not align with the developers' vision for their game.
+
+When Todd Howard, the executive producer for Bethesda was asked why some companies are not making their games modifiable, he replied, ``I don't understand why they don't, I think it makes your games better'' \cite{todd2012}. The \textit{Fallout 3} mod website Nexus hosts more than 7500 mods \cite{bostan2010explorations}, ranging from sexually revealing armours, nude female models, and the ``killable children'' mod \cite{bostan2010explorations}. The developers had already decided that the child characters in their game could not be harmed, so by enabling this feature the modders are directly contradicting the developers' ethics. Perhaps this subverting of a given design is exactly what a modification should be. But at what cost? A \textit{GTA V} developer famously cost Take-Two {\$}24 million when the hidden `hot-coffee' minigame was discovered causing suppliers to remove the game from sale \cite{hotcoffee1}.
```

Comment:
Consider re-positioning parts of this paragraph in the introduction. The opening gambit, quoting Todd Howard, has a stronger relation to the challenge your work strives to address. It will aid flow.